src/network/networkd-link.c

```diff
@@ -1660,6 +1662,11 @@ static int link_acquire_conf(Link *link) {
 
         assert(link);
 
+        if (link->setting_mtu) {
+                link->setting_mtu = false;
```

Comment:
it would be really good, to have a comment here that explains what this is about!